Sources/Plasma/PubUtilLib/plInputCore/plSceneInputInterface.cpp

```diff
@@ -545,9 +544,9 @@ bool    plSceneInputInterface::MsgReceive( plMessage *msg )
     plNotifyMsg* pNMsg = plNotifyMsg::ConvertNoRef(msg);
     if (pNMsg)
     {
-        for(int x=0; x < pNMsg->GetEventCount();x++)
+        for(size_t x = 0; x < pNMsg->GetEventCount(); x++)
         {
-            proEventData* pED = pNMsg->GetEventRecord(0);
+            proEventData* pED = pNMsg->GetEventRecord(x);
```

Comment:
:trollface:

Reply:
lolwut

Reply:
It's another case of attempting to iterate over a range but not actually using the iterator variable...